lib/jsdom/living/events/PointerEvent.webidl

```diff
@@ -0,0 +1,27 @@
+[Constructor(DOMString type, optional PointerEventInit eventInitDict), Exposed=Window]
```

Comment:
Could you add a comment with a link to https://w3c.github.io/pointerevents/#pointerevent-interface? Also, we generally copy the IDL over verbatim, so no need to collapse the whitespace :)

Reply:
Done